scripts/table_user_account_migration.py

```diff
@@ -18,13 +18,13 @@
     # ('Plan_Participants_and_Providers', 'EASI-2613', SQL_VARIANT_BASE_TASK_LIST),
     # ('Plan_Payments', 'EASI-2614', SQL_VARIANT_BASE_TASK_LIST),
     # ('Plan_Beneficiaries', 'EASI-2616', SQL_VARIANT_BASE_TASK_LIST),
-    ('Plan_General_Characteristics', 'EASI-2617', SQL_VARIANT_BASE_TASK_LIST),
+    # ('Plan_General_Characteristics', 'EASI-2617', SQL_VARIANT_BASE_TASK_LIST),
 
     # Base Struct
     # ('plan_cr_tdl', 'EASI-2618', SQL_VARIANT_BASE_STRUCT),
-    ('plan_discussion', 'EASI-2619', SQL_VARIANT_BASE_STRUCT),
+    # ('plan_discussion', 'EASI-2619', SQL_VARIANT_BASE_STRUCT),
     # ('discussion_reply', 'EASI-2619', SQL_VARIANT_BASE_STRUCT),
-    # ('plan_document', 'EASI-2620', SQL_VARIANT_BASE_STRUCT),
+    ('plan_document', 'EASI-2620', SQL_VARIANT_BASE_STRUCT),
     # ('plan_document_solution_link', 'EASI-2621', SQL_VARIANT_BASE_STRUCT),
     # ('operational_need', 'EASI-2623', SQL_VARIANT_BASE_STRUCT),
     # ('operational_solution', 'EASI-2624', SQL_VARIANT_BASE_STRUCT),

```

Comment:
Probably not best practice to update the tool script as part of the feature deployment but this is also sort of the fault of the tool for not using an external configuration. Just wanted to flag this so we establish a working agreement that it's ok to make changes in this script to support generation.